docs/features/multiple-networks/multi-network-policies.md

```diff
@@ -42,13 +67,13 @@ spec:
     matchLabels:
       app: stuff-doer # the policy will **apply** to all pods with this label
   ingress:
-  - ports:
-    - port: 9000
-      protocol: TCP
-    from:
-    - namespaceSelector:
-        matchLabels:
-          role: trusted # only pods on namespaces with this label will be allowed on port 9000
+    - ports:
+        - port: 9000
+          protocol: TCP
+      from:
+        - namespaceSelector:
+            matchLabels:
+              role: trusted # only pods on namespaces with this label will be allowed on port 9000
```

Comment:
I don't understand why you're changing the formatting; the above is valid yaml. It feels opinionated.

Reply:
I don't remember changing it. Perhaps it's the IDE? anyway changed it back